nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1530,16 +1556,9 @@ class FindRequest(BaseSearchRequest):
             SearchOptions.SEMANTIC,
         ]
     )
+    rank_fusion: Union[RankFusionName, RankFusion] = SearchParamDefaults.rank_fusion.to_pydantic_field()
```

Comment:
why not just `RankFusion`?

Reply:
To make the API cleaner. You can call by name and expect default parameters or specify them (if they have).
You can be shorter with
```
 {"rank_fusion": "legacy"}
```
instead of
```
{"rank_fusion": {"name": "legacy"}}
```
And you can be explicit and use parameters if you need to:
```
{"rank_fusion": {"name": "rrf", "k": 50}}
```